### ISSUE TITLE
Cannot read property 'isFunction' of undefined

### ISSUE THREAD

**Issue by HerrimanCoder** [avrahamgoldman/netflix.js#1]
Hello avigoldman,

I am very interested in your netflix.js module, but cannot get it to work. I tried pasting the code into the Chrome dev tools console, just as you stated. I did it 1 line at a time. All lines worked with no errors until I got here:

`head.appendChild(jqueryScript);`

That threw the error: **Cannot read property 'isFunction' of undefined**

And then I tried `flix.player.pause();` and I get `flix is not defined`.

I don't know if this is related or not, but I'm fairly certain that Netflix changed something significant in their object model in the past day or two. I used to be able to achieve seeks consistently by doing this:

`netflix.cadmium.UiEvents.events.resize[1].scope.events.dragend[idx].handler(null, {value: 300, pointerEventData: {playing: true}});`

That breaks now, because the _cadmium_ object no longer exists and ends up null.

In case it matters, I tried your code against **Armageddon on Netflix**.

Do you have success still with your code, as of today? If not, do you know how to make it work again? I am very anxious to learn.

**Comment by avigoldman** [comment]
Hey @HerrimanCoder I haven't tested this out for a bit of time now. Let me see if anything jumps out as wrong and I'll let you know ASAP!

**Comment by avigoldman** [comment]
Yeah, it looks like there are some bugs now. I'll comment to let you know when I have a fix implemented.

**Comment by HerrimanCoder** [comment]
Thank you...anxiously awaiting. :-)
